### ISSUE TITLE
[Feature Request] Config option to set default chapter sort order

### ISSUE THREAD

**Issue by qu1ck** [Suwayomi/Suwayomi-WebUI#790]
## What feature should be added to Suwayomi?
At some point recently the default sort order switched from reverse to normal. I would like an option to switch it back because scrolling all the way down on every manga or setting sort order for each manga individually is lame.

##  Why/Project's Benefit/Existing Problem
It's more useful to see the last chapters on top instead of at the bottom of a long scroll list. If it's not the default the should at least be an option to set that sort order as default.

**Comment by schroda** [comment]
> At some point recently the default sort order switched from reverse to normal.
this should not be the case.
I've just tested it with a new manga that hasn't been loaded before and the chapter list is sorted by source in descending order by default

try clearing the browser cache for suwayomi, this will clear the saved manga chapter list options

**Comment by qu1ck** [comment]
Well, somehow my local storage got filled with setting for each manga disabling reverse sort

![image](https://github.com/user-attachments/assets/4dacaf18-c998-486c-982d-1c60e132fb1a)

There might be a bug in settings migration somewhere? Either way, clearing the storage helped.

**Comment by schroda** [comment]
yeah, the bug is that I forgot about the stored chapter list filter settings when I made some changes 😅
you probably haven't cleared your browser cache in a while, so these stored settings are still from before those changes, which changed the default `reverse` value to `true` instead of `false`

**Comment by aoba-seragakii** [comment]
any update on this?

**Comment by Boomob** [comment]
Update?
